### DISCUSSION TITLE
i cannot find the protal to join the github education

### DISCUSSION THREAD

**Discussion by yummy08** [community/community#139238] [GitHub Education]
### Hi everyone,

<img width="1487" alt="image" src="https://github.com/user-attachments/assets/d2afc017-38c0-476e-bc50-284579689932">
When I click "join the GitHub Education", the web jumps to this page, and I cannot find the application form.

**Comment by yummy08** [comment]
It is so wired

**Comment by davevad93** [comment]
Hey @yummy08 ,

I don't know why this happens, but you aren't the first one experiencing this: #131668 , #135388 , #135546 

Try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem. This user #136400 solved it by doing that.

**Reply by yummy08** [reply]
Thank you very much, I have submitted a ticket.

**Reply by davevad93** [reply]
You're welcome @yummy08 , hope they help you to solve it as soon as possible! 👍

**Comment by AESMatias** [comment]
I've you tried https://education.github.com/discount_requests/application ?

**Reply by yummy08** [reply]
Yes， I've tried this URL, but it is redirected to the pricing page.

**Comment by Injamul2k24** [comment]
Try here  . if can be resolve 👍 
https://education.github.com/learner/learn
